bundler/lib/dependabot/bundler/native_helpers.rb

```diff
@@ -17,7 +17,6 @@ def self.run_bundler_subprocess(function:, args:, bundler_version:)
               # Bundler will pick the matching installed major version
               "BUNDLER_VERSION" => bundler_version,
               "BUNDLE_GEMFILE" => File.join(versioned_helper_path(bundler_version: bundler_version), "Gemfile"),
-              "BUNDLE_PATH" => File.join(versioned_helper_path(bundler_version: bundler_version), ".bundle"),
```

Comment:
This causes issues when running `bundle exec`, pretty sure we added this before `GEM_HOME` which ended up being the real fix for gem caches not being writeable by the `dependabot` user the default location was outside of the home directory